src/utils/hooks/api/use-get-vault-overview.tsx

```diff
@@ -1,80 +1,73 @@
-import { useQueries, UseQueryResult } from 'react-query';
+import { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-import { AccountId } from '@polkadot/types/interfaces';
-import Big from 'big.js';
-import {
-  CollateralIdLiteral,
-  tickerToCurrencyIdLiteral,
-  newAccountId,
-  VaultExt,
-  CurrencyIdLiteral
-} from '@interlay/interbtc-api';
-import { BitcoinUnit } from '@interlay/monetary-js';
+import { useQueries, UseQueryResult } from 'react-query';
+import { useErrorHandler } from 'react-error-boundary';
+import { newAccountId } from '@interlay/interbtc-api';
 
-import { HYDRA_URL } from '../../../constants';
-import issueCountQuery from 'services/queries/issue-count-query';
 import { useGetVaults } from 'utils/hooks/api/use-get-vaults';
 import { StoreType } from 'common/types/util.types';
-import { VAULT_WRAPPED } from 'config/vaults';
 
-interface VaultOverview {
-  apy: Big;
-  collateralization: Big | undefined;
-  issues: number;
-  collateralId: CurrencyIdLiteral;
-  wrappedId: CurrencyIdLiteral;
-}
+import { getVaultOverview, VaultData } from './get-vault-overview';
 
-const getVaultOverview = async (
-  vault: VaultExt<BitcoinUnit>,
-  accountId: AccountId
-): Promise<VaultOverview> => {
-  const tokenIdLiteral = tickerToCurrencyIdLiteral(vault.backingCollateral.currency.ticker) as CollateralIdLiteral;
-
-  const apy = await window.bridge.vaults.getAPY(accountId, tokenIdLiteral);
-  const collateralization = await window.bridge.vaults.getVaultCollateralization(accountId, tokenIdLiteral);
+type VaultTotals = {
+  totalLockedCollateral: number;
+  totalUsdRewards: number;
+  totalAtRisk: number;
+};
 
-  const issues = await fetch(HYDRA_URL, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json'
-    },
-    body: JSON.stringify({
-      query: issueCountQuery(`vault: {accountId_eq: "${accountId.toString()}"}, status_eq: Pending`)
-    })
-  });
+interface VaultOverview {
+  vaults: Array<VaultData> | undefined;
+  totals: VaultTotals | undefined;
+}
 
-  const issuesCount = await issues.json();
+const getVaultTotals = (vaults: Array<VaultData>) => ({
+  totalLockedCollateral: vaults.reduce((a, b) => a + b.collateral.usd, 0),
+  totalUsdRewards: vaults.reduce((a, b) => a + b.governanceTokenRewards.usd + b.wrappedTokenRewards.usd, 0),
+  totalAtRisk: vaults.map((vault) => vault.vaultAtRisk).filter(Boolean).length
```

Comment:
@peterslany going to merge this PR, but please answer this anyway when you have time :) I'm working on this hook for the business logic for the vaults table, so this can always be changed as part of that piece of work

Reply:
Morning @tomjeatt Sorry, I overlooked this on Friday. What I imagined was something like: 
`vaults.reduce((total, vault) => vault.atRisk ? total + 1 : total, 0)`  
What do you think about it? :)

Reply:
I've also tweaked the other instances of reduce to use `(total, vault)` instead of `(a, b)`